readme.md

```diff
@@ -144,6 +138,17 @@ Default: `latest`
 
 Which [dist-tag](https://docs.npmjs.com/adding-dist-tags-to-packages) to use to find the latest version.
 
+### notifier.checkImmediately()
+
+Check update immediately. This method will not trigger the checking process.
```

Comment:
I think is incorrect; This function does "trigger the checking process" since it calls and awaits https://github.com/sindresorhus/latest-version

Also `checkImmediately` sounds like it's a synchronous call but it returns a promise; perhaps it should be called `checkNow` or `fetchInfo`

Other than this, is this PR done?

Reply:
> I think is incorrect; This function does "trigger the checking process" since it calls and awaits https://github.com/sindresorhus/latest-version

My bad, I mean the child process:

```diff
- Check update immediately. This method will not trigger the checking process.
+ Check update immediately. This method will not trigger the child process.
```

> It should be called `checkNow` or `fetchInfo`.

I think `fetchInfo` is better.

> Other than this, is this PR done?

Yes, it's done.

Reply:
> My bad, I mean the child process:

I don't think that's useful information, I wouldn't particularly care about the inner workings of a method as long as it's clear about what it returns and when (and whether it triggers any network requests)

Reply:
Good point. Let me update the documentation.